WIP - trying to make API calls in React

chompasina opened the pull request:
- Adds Axios to make HTTP requests
- Adds Basic styling
- Trying to make API calls, but running into errors so it's commented out for now

Feedback requested on Axios implementation, specifically: 
- How to render multiple components (do they all need to be connected to app.js?)
- How to store API response in local storage
- How to secure API key in React (I'm pretty sure there's not really a way)

chompasina committed:
WIP - trying to make API calls in React

chompasina commented:
This is what is looks like now: 
<img width="887" alt="screen shot 2016-10-27 at 2 20 26 pm" src="https://cloud.githubusercontent.com/assets/15034459/19783827/a224c882-9c50-11e6-81f3-9ea3dd014d3c.png">

matthewrpacker commented on src/App.js:
```diff
@@ -2,19 +2,28 @@ import React, { Component } from 'react';
   zipsFromStorage = JSON.parse(zipsFromStorage);
 
-  this.setState({zips: zipsFromStorage ? zipsFromStorage : []});
+  this.setState({zips: zipsFromStorage ? zipsFromStorage : [], results: []});
```
@chompasina I really like the use of the conditional(ternary) operator in this case.  It is easy to understand and makes for a good one line conditional.

chompasina replied:
Thanks Matt! It's gotten more complicated now, but that was helpful feedback. I'll refactor so it can look like this again. :)

matthewrpacker commented on src/InputSubmission.js:
```diff
@@ -24,7 +24,7 @@ class InputSubmission extends Component {
              /><br/>
 
-          <button onClick={ this.handleClick.bind(this) } >Submit</button>
+          <button id="button" onClick={ this.handleClick.bind(this) } >Submit</button>
```
@chompasina Great job incorporating `.bind`.  I saw that you utilized it elsewhere, too.  Nice work!

chompasina committed:
Fixes API request, but need to fix CORS error

chompasina committed:
Refines zip search to Hillary or Trump:
- removes zipslist and fetchresults as components since storage is unnecessary
- adds ability to render committee and total information based on zip code submission

chompasina committed:
Refactors user input API calls to hit correct endpoint:
- User enters Zipcode and selects Trump, Hillary, or both to see individual contributions in zipcode
- The contributions are aggregated so I may want to pull in all of the contributions and tally them on the page
- Goal for tomorrow is to add visualizations to this data
